internal/storage/storageutil/object_util_test.go

```diff
@@ -0,0 +1,101 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	. "github.com/jacobsa/ogletest"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+const TestObjectName string = "gcsfuse/default.txt"
+
+func TestObjectAttrsToBucketObject(t *testing.T) { RunTests(t) }
+
+type objectAttrsToBucketObject struct {
+}
+
+func init() { RegisterTestSuite(&objectAttrsToBucketObject{}) }
+
+func (t objectAttrsToBucketObject) TestObjectAttrsToBucketObjectMethod() {
+	var attrMd5 []byte
+	Time := time.Now()
+	attrs := storage.ObjectAttrs{
+		Bucket:                  TestBucketName,
+		Name:                    TestObjectName,
+		ContentType:             "",
+		ContentLanguage:         "",
```

Comment:
by default string parameters will be empty. i dont think we need to explicitly set them as "". Same for booleans, they will be false.

Please set some values, so we are actually testing the method.

Reply:
fixed